packages/blueprints-integration/src/content.ts

```diff
@@ -52,67 +41,47 @@ export type SomeContent =
 	| CameraContent
 	| RemoteContent
 	| ScriptContent
-	| GraphicsContent
 	| NoraContent
 	| SplitsContent
-	| AudioContent
-	| CameraMovementContent
-	| LowerThirdContent
 	| LiveSpeakContent
-	| MicContent
 	| TransitionContent
 
 export interface VTContent extends BaseContent {
 	fileName: string
 	path: string
-	firstWords: string
-	lastWords: string
```

Comment:
I think I'm against removing firstWords/lastWords in a VTContent type, and would suggest just making them optional. The fact that we don't use that at the moment doesn't, in my opinion, make it redundant. It's a useful piece of information to the user and it should be assigned by the blueprints.

Reply:
I second this

Reply:
See the general comment below.

This one is a bit of an exception as I know there are plans to use it shortly, but would it be bad to remove the property now and then add it back when it does something?